scripts/native-system-benchmark.sh

```diff
@@ -46,80 +46,106 @@ EOF
 
 _help=
 if [[ $# -eq 0 ]]; then
-    _help=1
+	_help=1
 fi
 
 _err=0
 while [[ $# -gt 0 ]]; do
-    optarg=
-    shft_cnt=1
-    if [[ "$1" =~ [=] ]]; then
-        optarg="${1#*=}"
-    elif [[ "$1" =~ ^-- && $# -gt 1 && ! "$2" =~ ^- ]]; then
-        optarg="$2"
-        shft_cnt=2
-    elif [[ "$1" =~ ^-[^-] && $# -gt 1 && ! "$2" =~ ^- ]]; then
-        optarg="$2"
-        shft_cnt=2
-    elif [[ "$1" =~ ^-[^-] ]]; then
-        optarg="${1/??/}"
-    fi
-
-    case "$1" in
-        -s*|--samples*) DURATION="${optarg:-$DURATION}"; shift "$shft_cnt";;
-        --leak-check)  RECORD=debug; DBG="$VALGRIND"; shift "$shft_cnt";;
-        --debug)       RECORD=debug; shift "$shft_cnt";;
-        --profile)     RECORD=perf; shift "$shft_cnt";;
-        --clean-tests)
-            printf '%s\n' 'Deleting all test directories'
-            rm -rf -- "$BLD"/test-*; shift "$shft_cnt";;
-        --clean-seeds)
-            printf '%s\n' 'Deleting all cached preseeds'
-            rm -rf -- "$BLD"/preseeds; shift "$shft_cnt";;
-        --clean)
-            printf '%s\n' 'Deleting all tests and preseeds'
-            rm -rf -- "$BLD"/test-* "$BLD"/preseeds; shift "$shft_cnt";;
-        -d*|--debugger*)
-            case "$optarg" in
-                gdb) DBG="$GDB";;
-                rr)  DBG="$RR";;
-                *)   DBG="$optarg";;
-            esac
-            shift "$shft_cnt";;
-        -c*|--config*)
-            if [[ "$optarg" = /* ]]; then
-                ORIG_CFG="${optarg}"
-            else
-                ORIG_CFG="$CWD/$optarg"
-            fi
-            shift "$shft_cnt";;
-        -h|--help) _help=1; shift "$shft_cnt";;
-        *)
-            printf 'Unrecognized option: %s\n' "$1"
-            _help=1; _err=1;
-            break;;
-    esac
+	optarg=
```

Comment:
None of the other files use tabs for indentation; is there a reason this portion has been changed to tabs / reformatted?

Reply:
ACK, unintentional, reverted